src/unit.js

```diff
@@ -0,0 +1,38 @@
+define([
+	"./core",
+	"./unit/format",
+
+  "./plural"
+], function( Globalize, unitFormat ) {
+
+/**
+ * Globalize.formatUnit( value, unit, options )
+ *
+ * @value [Number]
+ *
+ * @unit [String]:
+ *
+ * @options [Object]
+ * - form: [String] "long", "short" (default), or "narrow".
+ *
+ * Format units such as seconds, minutes, days, weeks, etc.
+ */
+Globalize.prototype.formatUnit = function( value, unit, options ) {
+  var pluralGenerator;
+
+	if ( typeof value !== "number" ) {
+		throw new Error( "Value is not a number" );
+	}
+
+	if ( !unit ) {
+		throw new Error( "Missing unit" );
+	}
+
+	pluralGenerator = this.pluralGenerator();
+
+	return unitFormat( value, unit, options, pluralGenerator, this.cldr, this );
+};
```

Comment:
I'm a little confused by this - is it right to say `unitFormatter` should be the only function that touches the cldr data, and that `src/unit/format.js` should operate on the already extracted content?

Here's an example of one approach I was looking at for this, let me know if it looks sound.
https://gist.github.com/alunny/c9777c4c171b986fad9f